docs/validator/validator-faq.md

```diff
@@ -14,32 +14,55 @@ Fisherman (or validator) - nodes that provide security by verifying the validity
 
 ## How do I become a validator?
 
-If you already have an account with sufficient amount of funds.
+You need an account with sufficient amount of funds in the wallet.
 You can follow docs [here](local-setup/running-testnet.md) how to become validators
 
 Specific steps:
-Create new key pair that will be used for staking for given account
-Start a node with this key pair in validator_key.json
+Create new key pair that will be used for staking for given account, and load it with the funds you want to put at stake
+Start a node with the new key pair stored in validator_key.json
```

Comment:
I would recommend formatting special words using backticks like this: 

Start a node with the new key pair stored in `validator_key.json`